src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,25 @@
+fun main(args: Array<String>) {
+    scenario(12,8)
+}
+
+fun scenario(numberOfCars : Int, capacity : Int) {
+    /*
+    * Creates a given a number of cars, every other car wants to drive
+    * A road network is created with the given capacity
+    * The road network is analyzed and it is printed whether a car is delayed or not
+    */
+    val listOfCars = ListOfCars()
```

Comment:
The `listOfCars` is a little weird. Why not simply create a `List<Car>`, then loop to create some cars (who don't know about that list), add them to the list and give it to the network?